### PR TITLE
Added the EOL notice

### PR DESCRIPTION
**Pull request by GoliathLabs**
Good day,
I just added the EOL update nagger package. This will later be used by devices with low flash / RAM, so their respective owners will eventually upgrade to a newer and more powerful device. Inspired by Freifunk Frankfurt & Freifunk Karlsruhe.

### CODE REVIEW

File: ffmuc-eol-ssid/luasrc/lib/gluon/upgrade/950-ffmuc-update-nagger [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+end
+
+uci:section('wireless', 'wifi-iface', 'dep_radio0', {
+	ifname = 'eolnotice0',
+	network = 'client',
+	disabled = '0',
+	device = 'radio0',
+	mode = 'ap',
+	macaddr = macaddr,
+	ssid = 'supportende.muenchen.freifunk.net',
```
Comment (krombel) [inline ffmuc-eol-ssid/luasrc/lib/gluon/upgrade/950-ffmuc-update-nagger]: This does not work as the max length of SSID strings is 32 characters (while this string are 33)
So I would suggest something with "ffmuc.net" as it is shorter than "muenchen.ffmuc.net"

**Review by krombel — CHANGES_REQUESTED** [review]
other SSID required to make it work. Else the next ffmuc-meeting should be used to discuss/approve this.

**Comment by grische** [comment]
@krombel rebased on top of current main and renamed the SSID according to the last meeting notes end of July

**Review by krombel — APPROVED** [review]
LGTM

**Pull request merged**